packages/Toast/src/Toast.js

```diff
@@ -94,7 +94,7 @@ const Toast = React.forwardRef((props, ref) => {
         {!isVisuallyHidden && <sc.IconStyled as={icons[kind]} kind={kind} aria-hidden />}
         <sc.Content>{children}</sc.Content>
         {hasCloseButton && !isVisuallyHidden && (
-          <sc.CloseButtonStyled isSemantic={false} onClick={handleClose} size={types.SMALL} />
+          <sc.CloseButtonStyled isSemantic={false} onClick={handleClose} size={types.MEDIUM} />
```

Comment:
Since you'll need to push one more commit anyway, let's go ahead and just remove the `size` prop – the default value is `MEDIUM` anyway I believe.